app/bundles/EmailBundle/Tests/Model/SendEmailToContactTest.php

```diff
@@ -65,15 +65,44 @@ class SendEmailToContactTest extends \PHPUnit_Framework_TestCase
         ],
     ];
 
+    private $copyRepoMock;
+
+    private $mockDispatcher;
+
+    private $emailModelMock;
+
+    private $factoryMock;
+
+    private $routerMock;
+
+    private $statRepository;
+
+    private $dncModel;
+
+    private $mockEm;
+
+    protected function setUp()
```

Comment:
Why did you added so many Mocks here? The class `SendEmailToContact` has no dependencies like this. Where is use `MauticFactory` class for example?